Assets/Mirage/Runtime/Sockets/Udp/NanoSockets/Scripts/NanoSockets.cs

```diff
@@ -0,0 +1,222 @@
+/*
```

Comment:
Shouldn't we use git submodules instead than copying the file manually?

Reply:
Adding a submodule to the repo will probably require some CI tweaks to also fetch the sumbodules when checking out etc. The same would go for anyone who would want to clone the repo. Also not sure about UPM. I do use the git submodules personally but I must say it's quite tedious.